### DISCUSSION TITLE
What do you all think about having a marketplace to plugins and “open data” with NocoDB?

### DISCUSSION THREAD

**Discussion by ghost** [nocodb/nocodb#7289] [Q&A]
Hey y'all!

For example, Google has a marketplace so developers create integrations. Instead of the Google team create integrations with other systems, they are independent developers who create integrations with other systems for other independent developers or companies.

For example, I wanted to have these integrations here

- Connect NocoDB with Google Sheets(import/export)
- Connect NocoDB with Standard Notes(export)
- Connect NocoDB with Notesnook(export)
... etc

***disclaimer***
1. I do not work for NocoDB and have no connection with the company. In my case, I’m a NocoDB user and would like to contribute with some ideas.
2. These designs or ideas are not intended to endorse any position of the company or mine, they are just  demonstrations about the feedbacks I read here.
3. I would like to contribute to this technical discussions.
4. I hope to make an interesting view for most people.
5. I’m not promoting Google Sheets, Standard Notes, Google, Google Team, Google Marketplace, Notesnook ... just referring in bibliographic terms to something that could be developed or not. 
6. I would like to contribute to this resource, but I don't know how to start, I know little about programming.
7. This is more of a question than a feature request.
8. I don't have any money, but I can help implement this feature if it's interesting for everyone here.
9. The sky is the limit for those who dream. Something I initially thought of would be to have a marketplace for plugins and open data. So developers and data science scientists can develop and publish things on the NocoDB platform.

***faq***
*Have you all ever thought about creating a Marketplace on NocoDB?*

**Comment by dstala** [comment]
Thanks for opening up this discussion @nichdemos! We do have some of these ideas in the pipeline, but not in our immediate roadmap. But we would definitely like to hear more on this thread from the community to give us some pointers to be considered when we define this thread.

**Reply by ghost** [reply]
Hi @dstala, how are you? thanks for comment here!
